### ISSUE TITLE
settings button not focusable with the accessibility API when account has no profile picture

### ISSUE THREAD

**Issue by albertotirla** [element-hq/element-x-android#4599]
### Steps to reproduce

1. From the login screen, log in with an account which doesn't have a profile picture, or clear it if it does have one
2. Using talkback but I suspect other assistive technologies relying on the accessibility API will behave the same, go to the top of the screen after sync is complete

#### What did you expect?

The screenreader should announce that I'm focused on an element, of type button, named account preferences, settings or something like that. I've never seen the label it's supposed to show, so I'm guessing here

#### What happened instead?

The first control in there is the search button, then the room filters implemented as toggle buttons

### Your phone model

N/A

### Operating system version

N/A

### Application version and app store

latest, updated today from fdroid, but either way N/A

### Homeserver

synapse, but most likely N/A

### Will you send logs?

No

### Are you willing to provide a PR?

No
